.github/workflows/_cdk.yml

```diff
@@ -6,6 +6,21 @@ on:
       cdk_subcommand:
         required: true
         type: string
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+      - converted_to_draft
+    branches:
+      - develop
+      - 165-gh-actions-vite
```

Comment:
is this line a test line of code? It doesnt need to live in dev or main, does it?